source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOFillerSpaceManager.mm

```diff
@@ -0,0 +1,171 @@
+//
+//  ACOPaddingHandler.mm
+//  AdaptiveCards
+//
+//  Copyright © 2021 Microsoft. All rights reserved.
+//
+//
+
+#import "ACOFillerSpaceManager.h"
+#import "ACOBaseCardElementPrivate.h"
+
+const CGFloat kFillerViewLayoutConstraintPriority = UILayoutPriorityDefaultLow - 10;
```

Comment:
>10 [](http://example.com/codeflow?start=81&length=2)

why has this value of 10 picked? does this value matter at all?

Reply:
constraints with this priority will be overridden if there are competing constraints. UIKit puts default low for its low priority constraints.